daemon/info_unix.go

```diff
@@ -233,8 +233,9 @@ func parseInitVersion(v string) (version string, commit string, err error) {
 func parseRuncVersion(v string) (version string, commit string, err error) {
```

Comment:
can we also return the runtime name here?

Reply:
Added the runtime parsed from `version` command. Docker info actually uses the default runtime configured.